README.md

```diff
@@ -74,6 +74,26 @@ See the [oauth2 docs][] for complete instructions on using that library.
 For API methods that require HTTP Basic Authentication, use the
 [`BasicAuthTransport`](https://godoc.org/github.com/google/go-github/github#BasicAuthTransport).
 
+GitHub Apps authentication can be provided by [ghinstallation](https://github.com/bradleyfalzon/ghinstallation).
+
+```go
+import "github.com/bradleyfalzon/ghinstallation"
+
+func main() {
+  // Shared transport to reuse TCP connections.
+  tr := http.DefaultTransport
+
+  // Wrap the shared transport for use with the integration ID 1 authenticating with installation ID 99.
+  itr, err := ghinstallation.NewKeyFromFile(tr, 1, 99, "2016-10-19.private-key.pem")
+  if err != nil {
+    log.Fatal(err)
+  }
+
+  // Use installation transport with client
+  client := github.NewClient(http.Client{Transport: itr})
+}
```

Comment:
Replace `}` with `// use client...`. This is an incomplete program anyway (it's missing a `package` clause), so leaving out the closing `}` is okay. I don't see why you'd want to include `}` and make it seem like `client` isn't used at all.